src/scalable_cuckoo_filter.rs

```diff
@@ -192,6 +193,33 @@ impl<T: Hash + ?Sized, H: Hasher + Clone, R: Rng> ScalableCuckooFilter<T, H, R>
         self.filters.iter().map(|f| f.bits()).sum()
     }
 
+    /// Returns the number of cuckoo filters.
```

Comment:
As `CuckooFilter` struct is not exposed to users, I think this description is inadequate.
(For example, in this description, it should explain the meaning of "cuckoo filter" and how it differs from the `ScalableCuckooFilter` struct.)